dymos/transcriptions/common/endpoint_conditions_comp.py

```diff
@@ -248,7 +248,11 @@ def _setup_time(self):
             self.declare_partials(of='time++', wrt='{0}_value:time'.format(loc), val=1.0)
             self.declare_partials(of='time++', wrt='final_jump:time', val=1.0)
 
-    def setup(self):
+    def configure_io(self):
+        """
+        I/O creation is delayed until configure so that we can determine the shape and units for
+        the states.
+        """
         self._setup_time()
```

Comment:
super nit-pick, but maybe these should be `_config_time` etc.

Reply:
Fixed